HotelReservation/HotelReservation/Hotels/HotelSystem.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using HotelReservation.ReservationSteps;
+
+namespace HotelReservation.Hotels
+{
+    public class HotelSystem
+    {
+        private readonly List<Hotel> _hotels = new List<Hotel>();
+        public int LastHotelId { get; private set; } = 1000;
```

Comment:
Wygląda na to, że nie posiadając żadnego hotelu dalej dostaje jakieś LastHotelId, które niczemu nie odpowiada

Reply:
Fixed